aries-site/src/pages/components/avatar.mdx

```diff
@@ -0,0 +1,111 @@
+import { BestPracticeGroup, Example } from '../../layouts';
+import {
+  AvatarAccessibilityExample,
+  AvatarCardExampleDo,
+  AvatarCardExampleDont,
+  AvatarDataTableExampleDo,
+  AvatarDataTableExampleDont,
+  AvatarPageHeaderExample,
+  AvatarSizePreviewExample,
+} from '../../examples';
+
+## Anatomy
+
+Avatars can be displayed as an image, initials, or a logo. They come in three different sizes:
+
+- `small`: 24x24px
+- `medium`: 48x48px
+- `large`: 96x96px
+
+The container of an Avatar can be circular or a rounded square based on the use case.
+
+- Use a circle Avatar when representing a person either through an image or their initials.
+- Use a rounded square Avatar when representing a company’s or entity’s logo. This can be achieved by passing `round='medium'`
+- Always round the edges of a square container when using an avatar.
```

Comment:
```suggestion
- Always use `round="medium"` for a rounded square avatar.
```

Reply:
Also, what about logos that don't really "fill up" the container? image AT&T which is round, it doesn't make sense to clip the logo does it? @KennyAtHPE

Reply:
If its a case-by-case situation, you might have a list of companies that would have inconsistent avatar roundings

Reply:
so I changed the `round` to be `medium` and switched it out with a couple icons to see how it handles round icons and got this.

<img width="612" alt="Screen Shot 2022-10-19 at 10 53 36 AM" src="https://user-images.githubusercontent.com/42451602/196754712-264d71c7-9618-4de6-8474-9df1f2a87c94.png">

Reply:
Thanks for these examples. I'm wondering for these specific use cases, referring to the one on the left, can the user just show the icon instead of using an Avatar?

Reply:
What about saying the width can be flexible for non-square logos but the height is consistent. so height would be 48px but width was flexible to the content?

or is avatar built in a way where it's always a square? i forget.

Reply:
Good suggestion, can we get clarification on this @britt6612 ?

Reply:
Avatar takes in whatever size and sets that for both width and height so its the same..

```
      height: avatarSize,
      width: avatarSize,

```

also in grommet we shy away from using `width` and `height` props 
```

      'Avatar should use `size` instead of `height` or `width` props',

```

so Im not sure what the best approach here would be unless we change on grommet side to let it be more flexible to have a user set height using the `size` but the width can be changed

Reply:
yeah that may help some confusion. @KennyAtHPE I can set up a time today